UI/EncyclopediaDetailPanel.cpp

```diff
@@ -137,11 +137,26 @@ namespace {
         static std::vector<std::string> dir_names {
             "ENC_INDEX",        "ENC_SHIP_PART",    "ENC_SHIP_HULL",    "ENC_TECH",
             "ENC_BUILDING_TYPE","ENC_SPECIAL",      "ENC_SPECIES",      "ENC_FIELD_TYPE",
-            "ENC_METER_TYPE",   "ENC_EMPIRE",       "ENC_SHIP_DESIGN",  "ENC_SHIP",
-            "ENC_MONSTER",      "ENC_MONSTER_TYPE", "ENC_FLEET",        "ENC_PLANET",
-            "ENC_BUILDING",     "ENC_SYSTEM",       "ENC_FIELD",        "ENC_GRAPH",
-            "ENC_GALAXY_SETUP"};
-        //  "ENC_HOMEWORLDS" omitted due to weird formatting of article titles
+            "ENC_METER_TYPE",   "ENC_EMPIRE_DATA"};
+        return dir_names;
+    }
+
+    const std::vector<std::string>& GetEmpireDataDirNames() {
+        static std::vector<std::string> dir_names {
+            "ENC_EMPIRE",
+            "ENC_SHIP_DESIGN",
+            "ENC_SHIP",
+            "ENC_MONSTER",
+            "ENC_MONSTER_TYPE",
+            "ENC_FLEET",
+            "ENC_PLANET",
+            "ENC_BUILDING",
+            "ENC_SYSTEM",
+            "ENC_FIELD",
+            "ENC_GRAPH",
+            "ENC_GALAXY_SETUP",
+            "ENC_HOMEWORLDS"
```

Comment:
categorizing is fine, but a lot of this stuff seems like it's useful enough to have at the top tier listing of articles...